curriculum/challenges/english/08-coding-interview-prep/rosetta-code/k-d-tree.md

```diff
@@ -0,0 +1,354 @@
+---
+id: 5a23c84252665b21eecc7ecb
+title: K-d tree
+challengeType: 5
+---
+
+## Description
+<section id='description'>
+  A k-d tree (short for <i>k</i>-dimensional tree) is a space-partitioning data structure for organizing points in a k-dimensional space. 
```

Comment:
Could you remove the indent at the start of each line here? Doesn't make a difference in this problem, but could cause weird spacing if there's a `<pre>` section.